templates/class-live-stats.html

```diff
@@ -136,9 +136,25 @@ <h2 class="text-center">Class Overview</h2>
         <div class="grid grid-cols-2 gap-4 my-8" id="columns_grid">
             <!-- Card 2 Student List    -->
             <div id="col1">
-                <h2 class="text-center">
-                  Student List
-                </h2>
+                <div class="flex justify-center text-blue-500 space-x-3">
+                    <h2>
+                        Student List
+                    </h2>
+                    <div class ="relative">
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
+                            <path stroke-linecap="round" stroke-linejoin="round" d="M9.879 7.519c1.171-1.025 3.071-1.025 4.242 0 1.172 1.025 1.172 2.687 0 3.712-.203.179-.43.326-.67.442-.745.361-1.45.999-1.45 1.827v.75M21 12a9 9 0 11-18 0 9 9 0 0118 0zm-9 5.25h.008v.008H12v-.008z" />
+                        </svg>
+                        <div class='text-left tooltip opacity-0 hover:opacity-100 duration-300 absolute inset-0 z-10 flex justify-left items-top -mr-6 mt-0.5'>
+                            <span class='tooltiptext text-xs font-medium flex justify-left absolute rounded shadow-lg p-1 w-max' style="color:black"> 
+                                <ul style="list-style: none;"> 
+                                    <li><button type="button" class="h-3 w-3 border-2 border-blue-800 rounded-full"></button> - Current Adventure</li>
+                                    <li><button type="button" class="h-3 w-3 rounded-full bg-green-600"></button> - Quiz</li>
+                                    <li><button type="button" class="h-3 w-3 rounded-full bg-blue-500"></button> - Attempted</li>
+                                </ul>
```

Comment:
you could optionally include a fourth bubble that is gray: `h-6 w-6 m-1 rounded-full bg-gray-500` which says something like: `Not Attempted`